src/System.Threading.Timer/tests/TimerChangeTests.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+using Xunit;
+
+public class TimerChangeTests
+{
+    private static void EmptyTimerTarget(Object o) { }
+
+    [Fact]
+    public static void Timer_Change_NegativeTimeSpan_DueTime_Throws()
+    {
+        using (var t = new Timer(new TimerCallback(EmptyTimerTarget), null, 1, 1))
+        {
+            Assert.Throws<ArgumentOutOfRangeException>(new Action(() =>
+            {
+                t.Change(TimeSpan.FromMilliseconds(-2), new TimeSpan(1) /* not relevant */);
+            }));
+        }
+    }
+
+    [Fact]
+    public static void Timer_Change_NegativeTimeSpan_Period_Throws()
+    {
+        using (var t = new Timer(new TimerCallback(EmptyTimerTarget), null, 1, 1))
+        {
+            Assert.Throws<ArgumentOutOfRangeException>(new Action(() =>
+            {
+                t.Change(new TimeSpan(1) /* not relevant */, TimeSpan.FromMilliseconds(-2));
+            }));
+        }
+    }
+
+    [Fact]
+    public static void Timer_Change_NegativeInt_DueTime_Throws()
+    {
+        using (var t = new Timer(new TimerCallback(EmptyTimerTarget), null, 1, 1))
+        {
+            Assert.Throws<ArgumentOutOfRangeException>(new Action(() =>
+            {
+                t.Change(-2, 1 /* not relevant */);
+            }));
```

Comment:
FWIW, how you've written these is fine, but you shouldn't need as much ceremony, e.g. you should be able to write:
```C#
Assert.Throws<ArgumentOutOfRangeException>(() => t.Change(-2 1 /* not relevant */));
```
without explicitly constructing the Action delegate and with using an expression rather than than a statement block.